File: ProyectoMysql/server/BusinessLayer/Recepcion.py
from DataLayer.RecepcionDB import *
from EntityLayer.RecepcionEntity import *
from Utilidades.Conexion.configMysql import StartTransaction, EndTransaction, Restore


class Recepcion:

    def ObtenerMain():
        try:
            return RecepcionDB.ObtenerMain()
        except Exception as e:
            print(e)


    def ObtenerItem(Id : int):
        try:
            return RecepcionDB.ObtenerItem(Id)
        except Exception as e:
            print(e)

    def Registrar(Ent: RecepcionSaveModel) -> RecepcionSaveModel:
        try:
            StartTransaction()
            data = RecepcionDB.Registrar(Ent)
            EndTransaction()
            return data
        except Exception as e:
            Restore()
            print(e)
            

File: ProyectoMysql/Api/LogisticStorage/LogisticStorage.BusinessLayer/Recepcion.cs
using LogisticStorage.DataLayer;
using LogisticStorage.EntityLayer;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LogisticStorage.BusinessLayer
{
    public class Recepcion
    {


        public static List<RecepcionEntity> ObtenerMain()
        {
            RecepcionDB DB = new RecepcionDB();
            return DB.ObtenerMain();
        }
        public static List<RecepcionEntity> ObtenerItem(Int32 EmpresaId)
        {
            RecepcionDB DB = new RecepcionDB();
            return DB.ObtenerItem(EmpresaId);
        }
        public static Int32 Registrar(RecepcionEntity Ent)
        {
            RecepcionDB DB = new RecepcionDB();
            DB.Registrar(Ent);

            return Ent.RecepcionId;
        }
    }
}
